app/frontend/packs/degree-grade-autocomplete.js

```diff
@@ -15,6 +15,12 @@ const initDegreeGradeAutocomplete = () => {
       const container = document.getElementById(containerId);
       if (!container) return;
 
+      // Move autocomplete to the form group containing the input to be replaced
+      const inputFormGroup = container.previousElementSibling
+      if (inputFormGroup.contains(input)) {
+        inputFormGroup.appendChild(container)
+      }
```

Comment:
Could you place this into a `helpers.js` file and import and use it in all the necessary components?